packages/url/src/createEntryUrlBuilder.ts

```diff
@@ -54,9 +85,10 @@ export const createEntryUrlBuilder = ({ siteUrl, localize, homeUri }: EntryUrlPa
 			},
 			/**
 			 * Returns the entry uri relative to the language.
+			 * @deprecated Use `toSchemeLess` instead.
 			 */
 			toLanguageRelative() {
-				return uri;
+				return url;
```

Comment:
The return value is wrong here, maybe use `toSchemeless` for the value?

Reply:
@f-elix That's the old api, and it's deprecated. The builder is not language aware anymore.

Reply:
Ah right. 

Still, deprecating something means it still works. Now this is a breaking change, so I would just remove the function altogether instead.

Reply:
Good call. I've updated it 1aab47f4d5